app/pb_kits/playbook/pb_radio/radio.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbRadio
```

Comment:
I think all input kits should also function with an optional `block` to pass in a custom input. Example Toggle.  

Garett  is currently working on restructuring Card kit to define what "Block" kits should look like in new structure.

Reply:
I found the Card kit PR [here](https://github.com/powerhome/playbook/pull/322)